src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -701,5 +619,85 @@ internal override Diagnostic FilterDiagnostic(Diagnostic diagnostic)
         {
             return CSharpDiagnosticFilter.Filter(diagnostic, WarningLevel, GeneralDiagnosticOption, SpecificDiagnosticOptions);
         }
+
+        // 1.0 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
```

Comment:
Actually, if I change this to call the constructor with optional params, it should never need to be touched, including the body.